server/deployment/src/main/java/org/infinispan/quarkus/server/deployment/InfinispanServerProcessor.java

```diff
@@ -206,6 +205,11 @@ void addReflectionAndResources(BuildProducer<ReflectiveClassBuildItem> reflectio
       reflectionClass.produce(new ReflectiveClassBuildItem(false, false, ClusteredLockManagerConfigurationBuilder.class));
    }
 
+   @BuildStep
+   void excludeResourcesFromDependencies(BuildProducer<ExcludeConfigBuildItem> excludeConfigBuildItemBuildProducer) {
+      excludeConfigBuildItemBuildProducer.produce(new ExcludeConfigBuildItem("io.lettuce.lettuce-core-*.+.jar", "/META-INF/native-image/io.lettuce/lettuce-core/reflect-config.json"));
```

Comment:
```suggestion
      excludeConfigBuildItemBuildProducer.produce(new ExcludeConfigBuildItem("io.lettuce.lettuce-core-.+.jar", "/META-INF/native-image/io.lettuce/lettuce-core/reflect-config.json"));
```

I wonder if we should escape the dots also...? But if it works then it should be fine.